src/components/Kbd/Kbd.tsx

```diff
@@ -9,37 +9,33 @@ export interface KBDProps {
   separator?: string
 }
 
-const Kbd = forwardRef<HTMLElement, KBDProps>((props: KBDProps, ref) => {
-  const { kbds, separator } = props
+const Kbd = forwardRef<HTMLElement, KBDProps>(
+  ({ kbds = ['Ctrl', 'Shift', 'R'], separator = '+' }: KBDProps, ref) => {
+    const isLastKbd = React.useCallback(
+      (indexOfKey: number) => {
+        return kbds.length - 1 === indexOfKey
+      },
+      [kbds]
+    )
 
-  const isLastKbd = React.useCallback(
-    (indexOfKey: number) => {
-      return kbds.length - 1 === indexOfKey
-    },
-    [kbds]
-  )
-
-  return (
-    <p
-      className="text-info dark:text-gray-400"
-      ref={ref as LegacyRef<HTMLParagraphElement>}
-    >
-      {kbds.map((text, index) => (
-        <React.Fragment key={`${index}-text`}>
-          <kbd className="px-2 py-1.5 text-xs font-semibold text-gray-800 bg-gray-100 border border-gray-200 rounded-lg dark:bg-gray-600 dark:text-gray-100 dark:border-info">
-            {text}
-          </kbd>{' '}
-          <span>{!isLastKbd(index) && separator} </span>
-        </React.Fragment>
-      ))}
-    </p>
-  )
-})
+    return (
+      <p
+        className="text-info dark:text-gray-400"
+        ref={ref as LegacyRef<HTMLParagraphElement>}
+      >
+        {kbds.map((text, index) => (
+          <React.Fragment key={`${index}-text`}>
```

Comment:
From what I see this was already there, but we must avoid using the index as keys in the maps.